src/org-manager/components/organisation-details-info/organisation-details-info.component.html

```diff
@@ -1,5 +1,15 @@
 <ng-container *ngIf="org; else noOrg">
-  <h1 class="govuk-heading-xl">Organisation details</h1>
+    <ng-container *ngIf="org.status=='PENDING'; else showheading">
```

Comment:
The indenting here is off. The whole block needs to come back two spaces.

Reply:
and does it need triple ===?

Reply:
both the review comments addressed